projects/1.16/assets/hats/hats/lang/zh_cn.json

```
@@ -1 +1,244 @@
-{}
\ No newline at end of file
+{
+  "config.hats.cat.hatRandomisation.name": "帽子随机化",
+  "config.hats.cat.hatRandomisation.desc": "帽子随机化选项",
+  "config.hats.cat.others.name": "其他选项",
+  "config.hats.cat.others.desc": "不属于任何地方的选项。",
+
+  "config.hats.prop.maxHatRenders.name": "帽子最大渲染数量",
+  "config.hats.prop.maxHatRenders.desc": "每帧最多渲染的帽子数量",
+  "config.hats.prop.hatChance.name": "帽子生成概率",
+  "config.hats.prop.hatChance.desc": "对于客户端配置：在未安装帽子模组的服务器上，实体佩戴帽子的概率。\n对于服务器配置：支持佩戴帽子的实体佩戴帽子的概率。",
+  "config.hats.prop.hatUnlockString.name": "帽子解锁消息",
+  "config.hats.prop.hatUnlockString.desc": "解锁帽子、配饰时，屏幕提示的标题。0=随机，1=帽子已解锁，2=新帽子，3=帽子已获得，4=帽子已收集，5=得到了帽子",
+  "config.hats.prop.guiMinecraftStyle.name": "以Minecraft风格渲染GUI",
+  "config.hats.prop.guiMinecraftStyle.desc": "我们是否以Minecraft风格渲染GUI（而不是界面和其他东西）？1=原版风格，2=材质包风格",
+  "config.hats.prop.guiAnimationTime.name": "GUI动画时间",
+  "config.hats.prop.guiAnimationTime.desc": "GUI的大部分动画效果的持续时间（以刻为单位）。如果你想直接跳过动画效果，就设置为1。",
+  "config.hats.prop.shownTutorial.name": "展示GUI教程",
+  "config.hats.prop.shownTutorial.desc": "我们展示过GUI教程了吗？（设置为true以跳过它）",
+  "config.hats.prop.newHatToast.name": "新帽子弹窗",
+  "config.hats.prop.newHatToast.desc": "启用新帽子弹窗？",
+  "config.hats.prop.forceRenderToasts.name": "强制渲染弹窗",
+  "config.hats.prop.forceRenderToasts.desc": "华丽的帽子选择器GUI隐藏了游戏GUI。这禁用了弹窗的渲染。强制在我们的UI渲染器后面渲染弹窗？",
+  "config.hats.prop.forceGuiFallback.name": "强制使用备用GUI",
+  "config.hats.prop.forceGuiFallback.desc": "不喜欢选择帽子的时候旋转相机？打开这个选项以强制在屏幕内渲染实体，当然，打开GUI时按住潜行键也能强制这样做。",
+  "config.hats.prop.disableHatNameRenderInHatSelector.name": "在帽子选择器里禁用帽子名称渲染",
+  "config.hats.prop.disableHatNameRenderInHatSelector.desc": "卡顿？这个选项或许能提高一些帧数。",
+  "config.hats.prop.invisibleEntityInHatSelector.name": "在帽子选择器里隐藏玩家形象",
+  "config.hats.prop.invisibleEntityInHatSelector.desc": "卡顿？这个选项或许能提高一些帧数。",
+  "config.hats.prop.hatLauncherRandomHatSpeed.name": "帽子发射器-随机帽子循环速度",
+  "config.hats.prop.hatLauncherRandomHatSpeed.desc": "在帽子发射器的“随机帽子”选项中，每隔X刻就会显示一个新的帽子。增加这个数值可以减慢循环速度。",
+  "config.hats.prop.filterSorterConfig.name": "帽子排序配置",
+  "config.hats.prop.filterSorterConfig.desc": "帽子排序配置文件。请勿手动修改，通过帽子UI操作。",
+
+  "config.hats.prop.randSeed.name": "帽子随机化的随机种子。",
+  "config.hats.prop.randSeed.desc": "帽子随机化、稀有度、生成概率等的随机种子。留空则随机生成种子。",
+  "config.hats.prop.entityOverrideChance.name": "生物戴帽子几率",
+  "config.hats.prop.entityOverrideChance.desc": "生物戴帽子几率设置。每个条目由资源位置和几率组成，用逗号隔开。比如：“minecraft:pig,100” 就意味着所有的minecraft猪都百分百戴着帽子。",
+  "config.hats.prop.disabledMobs.name": "禁止戴帽子的生物",
+  "config.hats.prop.disabledMobs.desc": "用来禁用某些生物的帽子的资源位置，比如：minecraft:pig",
+  "config.hats.prop.rarityWeight.name": "稀有度权重",
+  "config.hats.prop.rarityWeight.desc": "每种帽子生成几率和稀有度权重，按Common, Uncommon, Rare, Epic, Legendary排序。任何缺少的值用0填充，多余的值丢弃。",
```

```suggestion
  "config.hats.prop.rarityWeight.desc": "每种帽子生成几率和稀有度权重，按Common、Uncommon、Rare、Epic、Legendary排序。缺少的值默认为0，多余的值会被丢弃。",
```
这个用的是原版的[稀有度](https://minecraft.fandom.com/zh/wiki/%E7%A8%80%E6%9C%89%E5%BA%A6)吗